data/functions.sql

```diff
@@ -1021,14 +1021,22 @@ BEGIN
       WHERE mu.wikidataid = wikidata_id;
   END IF;
 
-  -- finally, try states and provinces
+  -- try states and provinces
   IF NOT FOUND THEN
     SELECT
       min_label, max_label INTO min_zoom, max_zoom
       FROM ne_10m_admin_1_states_provinces sp
       WHERE sp.wikidataid = wikidata_id;
   END IF;
 
+  -- finally, try localities
+  IF NOT FOUND THEN
+    SELECT
+      rank_min, rank_max INTO min_zoom, max_zoom
```

Comment:
- `min_zoom` instead of `rank_min`
- There is no concept of `max_zoom` for NE populated places... so you can use whatever you want here. But probably it's better to use 0 (min min_zoom) or None or NULL or 21 (max possible zoom) rather than random values from the rank_max of the population.

Reply:
Or `ne_places_min_zoom` which is calculated from `scalerank` (which is odd).

Reply:
I will use min_zoom since scalerank hasn't been maintained for a while according to you.

Reply:
If you use `min_zoom` then you'd also need to change:

https://github.com/tilezen/vector-datasource/blob/master/queries.yaml#L35-L38

To have:

>      start_zoom: 2

Else the zoom 0/0/0 tile at 512px tile size will get a lot bigger because there are 11 features with min_zoom of `1.7`.

Reply:
My preference is to move to `min_zoom` instead of `ne_places_min_zoom` as `scalerank` isn't maintained for 5 years in Natural Earth and `min_zoom` is.